test/components/inputs/lInputLoaded.spec.ts

```diff
@@ -212,3 +212,30 @@ describe('returns empty placeholder when is outlined', () => {
     expect(inputLoaded.findComponent({name:'v-combobox'}).props().label).toBe('PlaceHolderText')
   })
 })
+
+describe('renders component when is disabled', () => {
```

Comment:
Acho que o describe novo é desnecessário, poderia ser só um case "sets input disabled style when disabled prop is true"

Reply:
Não deu boa tentei usando o setProps(). Mas como ele ja foi montado, não pega

Reply:
Então deixa o teste como "sets disabled style when disabled prop is true"